src/utils/use-users-handlers.js

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getUser } from '../redux/users/users.actions';
+
+const useUsersHandler = (id) => {
+  const dispatch = useDispatch();
+
+  const { user } = useSelector(({ Users }) => ({
+    user: Users.user
+  }));
+
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+
+  const [country, setCountry] = useState('');
+  const [city, setCity] = useState('');
+  const [adress, setAdress] = useState('');
+  const [postCode, setPostCode] = useState('');
+
+  const [ban, setBan] = useState('');
+
+  useEffect(() => {
+    dispatch(getUser(id));
+  }, [dispatch, id]);
+
+  useEffect(() => {
+    if (user !== null) {
+      setFirstName(user.firstName);
+      setLastName(user.lastName);
+      setCountry(user.address.country);
+      setCity(user.address.city);
+      setAdress(
+        `вул. ${user.address.street}, ${user.address.buildingNumber}/${user.address.appartment}`
+      );
+      setPostCode('79000');
```

Comment:
Why do we need to hardcode `postCode`?

Reply:
lt had been the really first commit, l fixed that